modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,246 @@
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config] 
+
+=== Data Loader
+
+`<graphql-router:data-loader>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration you use. | | *x*
+| Object Type Name | String | The object-type name. |  | *x*
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Primary Node Only | Boolean | Whether this source should only be executed on the primary node when running in cluster. |  | 
+| Redelivery Policy | xref:apikit-graphql-module-reference.adoc#redelivery-policy[Redelivery Policy] | Defines a policy for processing the redelivery of the same message. |  | 
+| Body Content | Any | Content of the message (usually, the payload). | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Array of Any
+|===
+
+==== For Configurations
+
+* xref:apikit-graphql-module-reference.adoc#config[Config] 
+
+== Types
+
+=== Data Fetcher Attributes
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Alias | String | Alias of the queried field. |  | false
+| Arguments | Object | Arguments of the queried field. |  | false
+| Selected Fields | Object | Map of subfields selected from this compound type. |  | 
+| Source | Any | Actual value of the fetched field parent. |  | false
+|======================
+
+=== Repeatable In Memory Stream
+
+|======================
+| Field | Type | Description | Default Value | Required
+| Initial Buffer Size | Number | This is the amount of memory that is allocated to consume the stream and provide random access to it. If the stream contains more data than buffer capacity, then it is expanded according to the bufferSizeIncrement attribute, with an upper limit of maxInMemorySize. |  | 
+| Buffer Size Increment | Number | This is by how much buffer size is expanded if it exceeds its initial size. Setting a value of zero or lower means that the buffer should not expand, meaning that a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised when the buffer gets full. |  | 
+| Max Buffer Size | Number | This is the maximum amount of memory used. If more is used, a STREAM_MAXIMUM_SIZE_EXCEEDED error is raised. A value lower or equal to zero means no limit. |  | 
+| Buffer Unit | Enumeration, one of:
+
+* BYTE
+* KB
+* MB
+* GB 
+
+| These attributes are expressed in this unit. |  | 
+|======================
+
+=== Repeatable File Store Stream
+
+|======================
+| Field | Type | Description | Default Value | Required
+| In Memory Size | Number | Defines the maximum memory that the stream should use to keep data in memory. If more is consumed, it starts to buffer the content on disk. |  | 
+| Buffer Unit | Enumeration, one of:
+
+* BYTE
+* KB
+* MB
+* GB 
+
+| The unit in which maxInMemorySize is expressed |  | 
```

Comment:
passive voice

Maybe:
The unit for maxInMemorySize

Reply:
OK! Thanks!